libs/transfer/src/lib/views/make-transfer-view/make-transfer-view.component.html

```diff
@@ -1,4 +1,4 @@
-<h1>{{ title }}</h1>
+<h1 *ngIf="title !== ''">{{ title }}</h1>
```

Comment:
- [ ] Please, add UT

Reply:
At the end I refactored this to an independent component called "MakeTransferTitleComponent" and made the unit tests there. So there is no need to make unit tests in the views anymore for this topic